packages/core/src/pipes/PipeSystem.ts

```diff
@@ -55,7 +55,15 @@ export class PipeSystem
         this.entryPath = options.entryPath;
     }
 
-    async transform(asset: Asset, pipeIndex = 0): Promise<void>
+    async transform(asset: Asset): Promise<void>
+    {
+        await this._transform(asset, 0);
+
+        // clean up any buffers still held for gc!
+        asset.releaseBuffers();
+    }
+
+    async _transform(asset: Asset, pipeIndex = 0): Promise<void>
```

Comment:
its a bit weird that `_transform` and `transform` exist together and both are marked as public

Reply:
good point! will make it private!